lib/manageiq/cross_repo/runner.rb

```diff
@@ -1,8 +1,13 @@
 require "manageiq/cross_repo/repository"
 require "active_support/core_ext/object/blank"
+Dir.glob(File.join(__dir__, "runner", "*")).sort.each { |f| require f }
 
 module ManageIQ::CrossRepo
   class Runner
+    def self.script_sources
+      constants.map { |c| const_get(c) }.grep(Class) - [Base]
```

Comment:
Could you use `Base.descendants` for this if you pull in `require "active_support/core_ext/class/subclasses"` ?

Reply:
Oh yeah, probably...let me try that

Reply:
I just dropped the entire script_sources method because there's no point to it once we switched to `Base.descendants`